doc/classes/String.xml

```diff
@@ -784,23 +861,25 @@
 		<method name="to_float" qualifiers="const">
 			<return type="float" />
 			<description>
-				Converts a string containing a decimal number into a [code]float[/code]. The method will stop on the first non-number character except the first [code].[/code] (decimal point), and [code]e[/code] which is used for exponential.
+				Converts the string representing a decimal number into a [float]. This method stops on the first non-number character, except the first decimal point ([code].[/code]) and the exponent letter ([code]e[/code]). See also [method is_valid_float].
```

Comment:
For these, I'm not sure about changing "containing" to "representing" since if I gather correctly, they can eat an input that's not a representation of a number, but contains one (i.e. "AAA 3.14 OOO")

Reply:
`to_int()` tries to remove all non-digit characters then do the conversion. `to_float()` **requires** the string to represent a decimal number.